homeassistant/components/cover/mqtt.py

```diff
@@ -58,6 +81,8 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
         config.get(CONF_NAME),
         config.get(CONF_STATE_TOPIC),
         config.get(CONF_COMMAND_TOPIC),
+        config.get(CONF_TILT_COMMAND_TOPIC),
+        config.get(CONF_TILT_STATUS_TOPIC),
```

Comment:
Can you add these to the schema? [Use these as a template](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/mqtt/__init__.py#L155), but tilt command should be optional.